f5_openstack_agent/lbaasv2/drivers/bigip/l7policy_service.py

```diff
@@ -90,7 +90,7 @@ def delete_l7policy(self, f5_l7policy, bigips):
                 LOG.error("L7 Policy deletion error: %s",
                           error.message)
 
-            return error
+        return error
```

Comment:
Is there a python linter that could have caught this problem (i.e., function did not call return in all cases)?

Reply:
Py.lint might have caught the error.